pkg/cluster/pod.go

```diff
@@ -304,9 +304,16 @@ func (c *Cluster) MigrateMasterPod(podName spec.NamespacedName) error {
 	}
 
 	masterCandidateName := util.NameFromMeta(masterCandidatePod.ObjectMeta)
-	if err := c.Switchover(oldMaster, masterCandidateName); err != nil {
-		return fmt.Errorf("could not failover to pod %q: %v", masterCandidateName, err)
-	}
+	_ = retryutil.Retry(1*time.Minute, 5*time.Minute,
```

Comment:
TODO try returning the error so that the new implementation matches the old behaviour of reportin non-movable master pod